### ISSUE TITLE
Document how to install and run program

### ISSUE THREAD

**Issue by jenniferlindberg** [andnil5/dd2480-grp12-assignment2#12]
Update README and add all details needed in order to run (and install) program.

**Comment by majate** [comment]
Important to mention that:

1. One needs to have a file with the path `src/env.py` including:
    ```py
    TOKEN='<github_token>'
    ```
    where `<gihub_token>` is a personal access token to the GitHub API.

2. One needs to have a copy of the repository, created by running the following command in the root directory:
    ```
    $ git clone <repo url> git_repo
    ```  
    *This makes sure the repo is put att the right place within a directory of the correct name (`git_repo`)*

**Comment by andnil5** [comment]
Fixed 1 in #26.

**Comment by andnil5** [comment]
Also, add instructions for how to install requirements. Something like "pip install -r requirements.txt" should work. :)
